odbc/src/api/data.rs

```diff
@@ -1078,6 +1078,30 @@ unsafe fn set_output_binary_helper(
 
 pub mod i16_len {
     use super::*;
+    ///
+    /// set_output_wstring_as_bytes writes [`message`] to the *WChar [`output_ptr`]. [`buffer_len`] is the
```

Comment:
Can you update the comment like you did for i32_len to point out the the buffer is a Byte buffer and not a Wchar buffer and sizes are Bytes sizes?